helm/hpcc/templates/thor.yaml

```diff
@@ -81,6 +86,9 @@ spec:
                 {{ printf "\"--name=%s\"" .name }}
               ]
 {{- include "hpcc.addSecurityContext" (dict "root" $ "me" .) | indent 8 }}
+{{- if not .useChildProcesses }}
+{{- include "hpcc.addStubResources" . | indent 8 }}
```

Comment:
perhaps should put a comment in here for the useChildProcess: true case, to say something like:
userChildProcesses=true in Thor is an experimental option only. It will cause manager and worker processes will be launched in this pod, and the pod would require enough resource for them all.